presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -285,6 +323,7 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                 new IterativeOptimizer(
                         stats,
                         statsCalculator,
+                        costCalculator,
```

Comment:
To make code copy-paste proof (can we assume optimizer blocks are never moved) you could
```
cc = estimatedExchangesCostCalculator
... all IterativeOptimizer using cc
addexchanges
cc = costCalculator
... all IterativeOptimizer using cc
```

This is unnecessary if `estimatedExchangesCostCalculator` chokes on exchanges and `costCalculator` verifies they are already there.